src/js/providers/html5.js

```diff
@@ -10,6 +10,9 @@ define([
 ], function(cssUtils, utils, stretchUtils, _, events, states, eventdispatcher, DefaultProvider) {
 
     var clearInterval = window.clearInterval,
+        stallInterval,
+        STALL_DELAY = 100,
+        STALL_TOLERANCE = STALL_DELAY/2,
```

Comment:
50 ms tolerance is too low (an animation lower than 20fps with no audio might not get time updates?). 125 ms (1/8 sec) would be much safer. It's slower than 1/16 -1/10 were the illusion of motion or delays are perceivable, but at least we won't send false stalls.